bin/spark-stats.js

```diff
@@ -1,39 +1,16 @@
+import '../lib/instrument.js'
```

Comment:
I really like the decision to move sentry code to its own file, this way it gets executed before the other imports